google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/wrappers/RowMutation.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.wrappers;
+
+import com.google.api.core.InternalApi;
+import com.google.bigtable.v2.MutateRowRequest;
+import com.google.bigtable.v2.MutateRowsRequest;
+import com.google.bigtable.v2.MutateRowsRequest.Entry;
+import com.google.bigtable.v2.TableName;
+import com.google.cloud.bigtable.data.v2.internal.RequestContext;
+import com.google.protobuf.ByteString;
+import javax.annotation.Nonnull;
+
+public class RowMutation extends AbstractMutation<RowMutation> {
+
+  private final String tableId;
+  private final ByteString key;
+
+  private RowMutation(String tableId, ByteString key) {
+    this.tableId = tableId;
+    this.key = key;
+  }
+
+  public static RowMutation create(@Nonnull String tableId, @Nonnull String key) {
+    return create(tableId, ByteString.copyFromUtf8(key));
+  }
+
+  public static RowMutation create(@Nonnull String tableId, @Nonnull ByteString key) {
```

Comment:
The current idiom is to use `of` for constructors of value types like this.

Reply:
I don't consider this a value type, it's more of a DSL/facade for the underlying proto. For the DSL, it makes more sense to create a mutation rather then to saying the mutation is of a row. 
I think
```
RowMutation.create("my-table", "my-key")
  .setCell("myfamily", "mycol", "my value")
```

Reads a lot better than
```
RowMutation.of("my-table", "my-key")
  .setCell("myfamily", "mycol", "my value")
```

Reply:
Ok, I can by that argument.